bliss/metrics.py

```diff
@@ -1,4 +1,8 @@
-from typing import Dict, List, Optional
+# pylint: disable=E1101
+# mypy: disable-error-code="union-attr"
```

Comment:
I personally haven't found it too useful, not sure about the others. I added this module-level suppression since mypy really doesn't play nice with union types - it kept complaining about the `Union[TileCatalog, FullCatalog]` not having certain attributes even though the type is checked in `update` to ensure that the catalog is the type I expect it to be.